oddbird/css-anchor-positioning
# Reduce built library size

## Description
This reduces the size of the built library from 223.72KiB to 105.54KiB by:

1. Switching to using more granular imports for csstree. 
2. Removing `source-map-js` from the built library. Source maps are still generated, but the module itself isn't needed.

Pre-merge cleanup:
- [ ] Remove bundle-stats plugin?

## Related Issue(s)
#276 


## Steps to test/reproduce

- Verify that the demo works locally, by building, and then changing the import in index.html to `import polyfill from '/dist/css-anchor-positioning-fn.js';`
- Locally, run `npm run build; open dist/bundle-stats.html` to check the results. (You can compare with the current build on the `prev-stats` branch, which has the bundle-stats plugin installed with no modifications to main.

## Show me

**Before**
<img width="1162" alt="image" src="https://github.com/user-attachments/assets/2a7dddb4-83f2-4bdf-a451-977f6c9401c3">

**After**
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/da252284-7806-45bf-9ad1-6bbbf5698dac">
